CONTRIBUTING.adoc

```diff
@@ -20,6 +20,53 @@ Thank you for your contributions! Please follow this process to submit a patch:
 NOTE: If you submit a new add-on, the add-on must be added as a sub-directory in the *_/add-ons/_* sub-directory of the repository.
 See the documentation for information about link:https://docs.openshift.org/latest/minishift/using/addons.html#add-ons-overview[file structure and content format].
 
+[[testing]]
```

Comment:
This should be in separate file and should referenced from README. Any additoin to `CONTRIBUTING.MD` will be shown as part of link during new issue creation.